### PR TITLE
Fix builds

### PR DESCRIPTION
**Pull request by AntonKozlov**
Fix builds in vairous configurations

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Reviewers
 * [Jan Kratochvil](https://openjdk.org/census#jkratochvil) (@jankratochvil - no project role)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/crac.git pull/104/head:pull/104` \
`$ git checkout pull/104`

Update a local copy of the PR: \
`$ git checkout pull/104` \
`$ git pull https://git.openjdk.org/crac.git pull/104/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 104`

View PR using the GUI difftool: \
`$ git pr show -t 104`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/crac/pull/104.diff">https://git.openjdk.org/crac/pull/104.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/crac/pull/104#issuecomment-1675880822)

### CODE REVIEW

File: src/hotspot/cpu/s390/vm_version_s390.hpp [hunk 1]
```diff
@@ -568,7 +568,7 @@ class VM_Version: public Abstract_VM_Version {
   static unsigned long z_SIGSEGV();
 
   // No _features_names[] available on this CPU.
-  static void insert_features_names(char* buf, size_t buflen, uint64_t features = _features) {}
+  static void insert_features_names(char* buf, size_t buflen, unsigned long features[] = _features) {}
```
Comment (jankratochvil) [inline src/hotspot/cpu/s390/vm_version_s390.hpp:571]: I do not see a reason for this change. How to test the s390 build?

Comment (AntonKozlov) [reply src/hotspot/cpu/s390/vm_version_s390.hpp:571]: From the line 123 from this file --  _features is 
```
  static unsigned long _features[_features_buffer_len];
```

The build is tested in GHA https://github.com/AntonKozlov/crac/actions/runs/5841005695/job/15841780175

Comment (AntonKozlov) [reply src/hotspot/cpu/s390/vm_version_s390.hpp:571]: Here the failure before the change: https://github.com/jankratochvil/crac/actions/runs/5854032342/job/15870802152

Comment (jankratochvil) [reply src/hotspot/cpu/s390/vm_version_s390.hpp:571]: Thanks, I see now GHA does it test. Yes, I agree your fix is correct.

Comment (AntonKozlov) [reply src/hotspot/cpu/s390/vm_version_s390.hpp:571]: Could you Approve the PR  then?

**Pull request closed without merging**